content/docs/bdd/_index.md

```diff
@@ -6,4 +6,107 @@ menu: main
+
+BDD helps teams to have the right conversations at the right time so you minimise the amount of time spent in meetings and maximising the amount of valuable code you produce.
+
+We use structured conversations, called [discovery workshops], that focus around real-world examples of the system from the users' perspective. These conversations grow our team's shared understanding of the needs of our users, of the rules that govern how the system should function, and of the scope of what needs to be done.
+
+It may also reveal gaps in our understanding, where we need more information before we know what to do.
+
+The scrutiny of a discovery session often reveals low-priority functionality that can be deferred the scope of a user story, helping to team to work in smaller increments and improving their flow.
+
+If you're new to BDD, exploration is the best place to start. It's useless to try the other two practices until you've mastered exploration.
+
+## Specification
+
+Taking the real-world examples of system behaviour from our discovery sessions, the technical folks on the team now try to express those examples as structured documentation. This gives us a quick way to confirm that their understanding of what to build matches what the business folks have envisaged.
+
+In contrast to traditional documentation, we use a medium that can be read by both humans and computers, so that:
+
+* We can get feedback from the business folks about whether we understand them correctly.
+* We'll be able to automate these examples to guide our development of the implementation.
+
+By collaborating with business folks to write this executable specification, we establish a shared language for talking about the system. This is really useful for programmers who are notoriously [bad at naming things](https://martinfowler.com/bliki/TwoHardThings.html).
+
+## Automation
+
+Now that we have our executable specification, we can use it to guide our development of the implementation.
+
+Taking one example at a time, we automate it by connecting it to the system as a test. The test fails because we have not implemented the behaviour it describes yet. Now we develop the implementation code, using [lower-level examples of the behaviour of internal system components](https://anarchycreek.com/2009/05/20/theyre-called-microtests/) to guide us as required.
+
+The automated examples work like guide-rails, helping us to keep our development work on track.
+
+When we need to come back and maintain the system later, the automated examples will help us to understand what the system is currently doing, and to make changes safely without unintentionally breaking anything.
+
+This rapid, repeatable feedback reduces the burden of manual testing, freeing people up to do more interesting work, like exploratory testing.
```

Comment:
```suggestion
This rapid, repeatable feedback reduces the burden of manual regression testing, freeing people up to do more interesting work, like exploratory testing.
```

Reply:
Possibly link to a good explanation of exploratory testing somewhere?

Reply:
Yes, actually I was planning to add a terminology page for that. Where do you think would be a good place to point people towards?